src/content/learn/separating-events-from-effects.md

```diff
@@ -136,13 +136,13 @@ export function sendMessage(message) {
 }
 
 export function createConnection(serverUrl, roomId) {
-  // A real implementation would actually connect to the server
+  // Gerçek bir uygulama sunucuya gerçekten bağlanır
   return {
     connect() {
-      console.log('✅ Connecting to "' + roomId + '" room at ' + serverUrl + '...');
+      console.log(serverUrl + ' + ✅ adresinde "' + roomId + '" odasina baglaniliyor' + '...');
     },
     disconnect() {
-      console.log('❌ Disconnected from "' + roomId + '" room at ' + serverUrl);
+      console.log(serverUrl + '❌ adresinde "' + roomId + '" odasının bağlantısı kesildi ' );
```

Comment:
Burada `❌` isareti cumlenin ortasina degil, basina gelmesi lazim.

Reply:
Hataları uyardığınız için teşekkür ederim. Yeni PR'da düzeltildi.